src/components/Gallery.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Album from '../components/Album';
+
+export default () => (
+  <div className="gallery-container">
+    <h1 className='title is-4'>Gallery</h1>
+    <h2 className='subtitle is-5'>Featured Albums</h2>
+    <div className='columns is-multiline albums'>
+      <Album title='Student Portfolio' />
```

Comment:
You could define an array with all the titles and then map over that. Don't repeat code

Reply:
Alright